src/main/java/elements/Autocomplete.java

```diff
@@ -0,0 +1,27 @@
+package elements;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Autocomplete {
+    //uiAutocomplete
+    WebDriver driver;
+    String label;
+    String locator = "//*[contains(text(),'%s')]/ancestor::div[contains(@class,'uiInput')]//input";
+    String selectFirstItem = "//*[contains(@class,'uiAutocomplete')]//ul/li[1]";
+
+    public Autocomplete(WebDriver driver, String lable) {
+        this.driver = driver;
+        this.label = lable;
+    }
+
+    public void write(String text) {
+        System.out.println(String.format("writing text: %s with label: %s", text, label));
+        driver.findElement(By.xpath(String.format(locator, label))).sendKeys(text);
+    }
+
+    public void select(String text) {
+        write(text);
+        driver.findElement(By.xpath(selectFirstItem)).click();
```

Comment:
tests are failed on this step (account and contact):

NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[contains(@class,'uiAutocomplete')]//ul/li[1]"}

maybe you need to put a waiter before clicking on this

Reply:
I added an waiter, but here the error occurs most likely due to an empty list